DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -246,45 +247,72 @@ elif [[ $ALIGNLEVEL == 1 ]]; then
+  if [[ $ALIEN_JDL_LPMANCHORYEAR == "2022" ]]; then
+    INST_IR_FOR_TPC="CTPCCDB"
+  fi
+  if [[ $PERIOD == "LHC22s" ]]; then
+    INST_IR_FOR_TPC=0 # in this way, only TPC/Calib/CorrectionMaps is applied, and we know that for 22s it is the same as TPC/Calib/CorrectionMapsRef
+  fi
+  # in MC, we set it to a negative value to disable completely the corrections
+  if [[ $ALIEN_JDL_LPMPRODUCTIONTYPE == "MC" ]]; then
+    INST_IR_FOR_TPC="-1"
+  fi
+
+  # now we set the options
+  if [[ $INST_IR_FOR_TPC -gt 0 ]]; then # externally imposed IR for scaling
+    echo "Applying externally provided IR for scaling, $INST_IR_FOR_TPC Hz"
+    export TPC_CORR_SCALING+=" --corrmap-lumi-inst $INST_IR_FOR_TPC "
+  elif [[ $INST_IR_FOR_TPC == 0 ]]; then # when zero, only the TPC/Calib/CorrectionMaps is applied
+    echo "Passed valued for scaling is zero, only TPC/Calib/CorrectionMaps will be applied"
+    export TPC_CORR_SCALING+=" --corrmap-lumi-inst $INST_IR_FOR_TPC "
+  elif [[ $INST_IR_FOR_TPC -lt 0 ]]; then # do not apply any correction
+    echo "Passed valued for scaling is smaller than zero, we will not apply any correction"
+    export TPC_CORR_SCALING+=" --corrmap-lumi-inst $INST_IR_FOR_TPC "
+  elif [[ $INST_IR_FOR_TPC == "CTPCCDB" ]]; then # using what we have in the CCDB CTP counters, extracted at the beginning of the script
+    echo "Using CTP CCDB which gave the mean IR of the run at the beginning of the script ($RUN_IR Hz)"
+    export TPC_CORR_SCALING+=" --corrmap-lumi-inst $RUN_IR "
+  elif [[ $INST_IR_FOR_TPC == "CTP" ]]; then
+    if ! has_detector CTP ; then
+      echo "TPC correction with CTP Lumi is requested but CTP is not in the WORKFLOW_DETECTORS=$WORKFLOW_DETECTORS"
+      return 1
     fi
-    TRACKTUNETPCINNER="$TRACKTUNEPARAMSDATAONLY;trackTuneParams.sourceLevelTPC=true;trackTuneParams.tpcCovInnerType=1;trackTuneParams.tpcCovInner[0]=0.025;trackTuneParams.tpcCovInner[2]=0.2e-4;trackTuneParams.tpcCovInner[3]=0.2e-4;trackTuneParams.tpcCovInner[4]=0.002;"
-    CUT_MATCH_CHI2=60
+    echo "Using CTP inst lumi stored in data"
+    export TPC_CORR_SCALING+=" --require-ctp-lumi "
+  else
+    echo "Unknown setting for INST_IR_FOR_TPC = $INST_IR_FOR_TPC (with ALIEN_JDL_INST_IR_FOR_TPC = $ALIEN_JDL_INST_IR_FOR_TPC)"
+    return 1
   fi
-  export ITSTPCMATCH="tpcitsMatch.safeMarginTimeCorrErr=10.;tpcitsMatch.cutMatchingChi2=$CUT_MATCH_CHI2;;tpcitsMatch.crudeAbsDiffCut[0]=6;tpcitsMatch.crudeAbsDiffCut[1]=6;tpcitsMatch.crudeAbsDiffCut[2]=0.3;tpcitsMatch.crudeAbsDiffCut[3]=0.3;tpcitsMatch.crudeAbsDiffCut[4]=5;tpcitsMatch.crudeNSigma2Cut[0]=100;tpcitsMatch.crudeNSigma2Cut[1]=100;tpcitsMatch.crudeNSigma2Cut[2]=100;tpcitsMatch.crudeNSigma2Cut[3]=100;tpcitsMatch.crudeNSigma2Cut[4]=100;"
+
+  if [[ -n $ALIEN_JDL_MEANIRFORTPC && $ALIEN_JDL_MEANIRFORTPC > 0 ]]; then # externally imposed TPC map mean IR for scaling
+    export TPC_CORR_SCALING+=" --corrmap-lumi-mean $ALIEN_JDL_INST_IR_FOR_TPC "
```

Comment:
there is a typo here, should be: 
```
    export TPC_CORR_SCALING+=" --corrmap-lumi-mean $ALIEN_JDL_MEANIRFORTPC "
```

Reply:
Yes, sorry, I had copied-pasted. 
Just FYI, when you use an "ALIEN_JDL" variable, all that comes after that should be 1 single word.